app/scripts/ui/zipcode.js

```diff
@@ -0,0 +1,24 @@
+/** @jsx React.DOM */
+var React   = require('react');
+var Modal   = require('react-bootstrap/Modal');
+var Button  = require('react-bootstrap/Button');
+var Input   = require('react-bootstrap/Input');
+
+var Zipcode = React.createClass({
+    displayName: 'Zipcode',
+    render: function(){
+        return(
+            <Modal title="Enter your zipcode">
+                <div className="modal-body" id="zipcode-body">
+                    <Input
```

Comment:
This we might need to change to ZipcodeInput component later, as the same is used in search page